LoRaEngine/LoraKeysManagerFacade/EdgeDeviceGetter.cs

```diff
@@ -28,18 +28,11 @@ public class EdgeDeviceGetter : IEdgeDeviceGetter
             this.logger = logger;
         }
 
-#pragma warning disable IDE0060 // Remove unused parameter. Kept here for future improvements of RegistryManager
-        private async Task<List<Twin>> GetEdgeDevicesAsync(CancellationToken cancellationToken)
-#pragma warning restore IDE0060 // Remove unused parameter
+        private async Task<IEnumerable<IDeviceTwin>> GetEdgeDevicesAsync(CancellationToken cancellationToken)
```

Comment:
We are trying to logically map Edge Devices to IDeviceTwin.

IDeviceTwin (now including both GatewayId and NwkSKey information) is more a "ILoRaDeviceTwin" rather than a generic IDeviceTwin (which ideally should be limited to just ETag, Properties, Tags and DeviceId)

Please either revert this change or, better yet, create a base IDeviceTwin with generic Twin information as above and extend that with a ILoRaDeviceTwin (including GatewayId and NwkSKey)